.all-contributorsrc

```diff
@@ -41,6 +41,15 @@
       "contributions": [
         "content"
       ]
+    },
```

Comment:
@abdenassar01 could you add another commit and remove your changes from this file.
The All contributors Bot does these changes automatically and will now cause a conflict